# -*- coding: utf-8 -*-
# Generated by Django 1.10 on 2016-09-08 15:54
from __future__ import unicode_literals

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('apps', '0031_auto_20160818_1057'),
    ]

    operations = [
        migrations.AlterField(
            model_name='article',
            name='url',
            field=models.URLField(blank=True, help_text='Внешний URL, по которому доступна статья, которой вы желаете поделиться.', null=True, unique=True, verbose_name='URL статьи'),
        ),
        migrations.AlterField(
            model_name='historicalarticle',
            name='url',
            field=models.URLField(blank=True, db_index=True, help_text='Внешний URL, по которому доступна статья, которой вы желаете поделиться.', null=True, verbose_name='URL статьи'),
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
